src/index.html

```diff
@@ -217,10 +217,15 @@ <h1 id="dialogTitle">Open File</h1>
       crossorigin="anonymous"
     />
     <script>
-      const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
-      if (timeZone === 'Asia/Shanghai' && window.location.hostname !== 'cnij.imjoy.io') {
-        document.getElementsByClassName('site-tips')[0].style.display = 'block';
-      }
+      setTimeout(() => {
+        const timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+        const hostnameArr = ['cnij.imjoy.io', '0.0.0.0', '127.0.0.1', 'localhost']
+        if (timeZone === 'Asia/Shanghai' && !hostnameArr.includes(window.location.hostname)) {
+          document.getElementsByClassName('site-tips')[0].style.display = 'block';
```

Comment:
To be more efficient, maybe we can assign an id to the site-tips element and use `document.getElementById`